protos/capture_data.proto

```diff
@@ -56,3 +56,7 @@ message LinuxAddressInfo {
   string function_name = 3;
   uint64 offset_in_function = 4;
 }
+
+message CaptureHeader {
+  uint32 version = 1;
```

Comment:
add/replace with string, we need something to show to users. (This capture was saved with v1.42 earliest supported version is v1.80...)